modules/ROOT/pages/getting-started/connect-instance.adoc

```diff
@@ -37,4 +37,58 @@ The default is `neo4j+s//`.
 
 |Password
 |You are given the password when you initially create the instance
-|===
\ No newline at end of file
+|===
+
+== Connection methods
+
+To connect to an instance, you select one of the following connection methods: Neo4j or Query API.
+
+*Neo4j*: `neo4j+s//` This is the default connection method. 
+This scheme is built on the `bolt://` protocol and uses a WebSocket connection to the Neo4j port. 
+Use for best performance and stability.
+
+*Query API*: Connects to Aura through the Query API via HTTPS on port 443 (a standard, widely open port, ensuring compatibility with most firewalls). 
+Use for debugging or when Bolt is unavailable due to your organization's network configuration.
+
+For more information about connection schemes, see link:https://neo4j.com/docs/operations-manual/current/configuration/connectors/[Operations Manual -> Configure network connectors] and link:https://neo4j.com/docs/bolt/current/bolt/[Bolt Protocol].
+
+== Troubleshooting
```

Comment:
```suggestion
=== Connection error
```
_Troubleshooting_ indicates a list of problems and their solutions, this is only one problem so it's better to just call it what it is. Also, it doesn't need a level 2 header.